generator/generator.go

```diff
@@ -99,6 +99,10 @@ var (
 	simplifyWildcardPaths   = flag.Bool("simplify_wildcard_paths", false, "Whether to omit the keys in the generated paths if all keys for a list node are wildcards.")
 	listBuilderKeyThreshold = flag.Uint("list_builder_key_threshold", 0, "The threshold equal or over which the path structs' builder API is used for key population. 0 means infinity. This flag is only meaningful when wildcard paths are generated.")
 	pathStructSuffix        = flag.String("path_struct_suffix", "Path", "The suffix string appended to each generated path struct in order to differentiate their names from their corresponding schema struct names.")
+	splitByModule           = flag.Bool("split_by_module", false, "Whether to split path struct generation by module.")
+	fakeRootPackageName     = flag.String("fake_root_package_name", "device", "Name for the fake root package, when spilting generation by module.")
+	trimOCPackage           = flag.Bool("trim_oc_package", false, "Whether to trim openconfig- from generated package names.")
+	baseImportPath          = flag.String("base_import_path", "", "Base import path used concatenated with module for path struct imports.")
```

Comment:
... used to concatenate with module package relative paths for ...

Reply:
fixed